packages/portalnetwork/src/networks/beacon/beacon.ts

```diff
@@ -51,6 +51,7 @@ import type { Debugger } from 'debug'
 import type { AcceptMessage, FindContentMessage, OfferMessage } from '../../wire/types.js'
 import type { ContentLookupResponse } from '../types.js'
 import type { BeaconChainNetworkConfig, HistoricalSummaries, LightClientForkName } from './types.js'
+import type { INodeAddress } from '@chainsafe/discv5/lib/session/nodeInfo.js';
```

Comment:
I'm really surprised this works since discv5 has an exports map.  Could we maybe just replicate this in our own types so we don't have to deal with a deep import like this?